pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -339,41 +348,59 @@ func (vm *windows) initializeBootstrapperFiles() error {
 }
 
 // createService creates the service on the Windows VM
-func (vm *windows) createService(svc service) error {
-	out, err := vm.Run("sc.exe create "+svc.Name()+" binPath=\""+svc.BinaryPath()+" "+
-		svc.Args()+"\" start=auto", false)
+func (vm *windows) createService(svc *service) error {
```

Comment:
Nit: ideally all functions that take a pointer input should have a nil pointer check.